yesod-core/src/Yesod/Core/Class/Handler.hs

```diff
@@ -19,7 +19,6 @@ import Control.Monad.Trans.Class (lift)
 import Data.Conduit.Internal (Pipe, ConduitM)
 
 import Control.Monad.Trans.Identity ( IdentityT)
-import Control.Monad.Trans.List     ( ListT    )
```

Comment:
This is a breaking change, and should not be included right now.

Reply:
Good point, closing.